packages/babel-plugin-proposal-pipeline-operator/src/fsharpVisitor.js

```diff
@@ -0,0 +1,27 @@
+import { types as t } from "@babel/core";
+import maybeOptimizePipelineSequence from "./maybeOptimizePipelineSequence";
+
+const fsharpVisitor = {
+  BinaryExpression(path) {
+    const { scope } = path;
+    const { node } = path;
```

Comment:
Nit: these two statements can be merged

Reply:
Done in ecbb465e2dc779ee632bf1d1664ed1b3d49e5565